src/openstudio_app/CMakeLists.txt

```
@@ -413,17 +382,21 @@ if( APPLE )
   list(APPEND QTPLUGINS "\${CMAKE_INSTALL_PREFIX}/OpenStudioApp.app/Contents/Frameworks/QtWebEngineCore.framework/Versions/5/Helpers/QtWebEngineProcess.app/Contents/MacOS/QtWebEngineProcess")
 endif()
 
-# This does some "magic" **including on Linux** (cf #345) (On linux it will install in ./bin/ the libopenstudio.so, and libicu* stuff)
-install(CODE "
-  # APPS might be a symbolic link which does not work with fixup_bundle
-  # expand APPS to a real path
-  get_filename_component(APPS_EXPANDED ${APPS} REALPATH)
-  include(BundleUtilities)
-  # This file defines some libs we are excluding. Some like concrt140.dll are installed already via InstallRequiredSystemLibraries
-  include(\"${CMAKE_SOURCE_DIR}/CMake/FixupBundleOverrides.cmake\")
-  fixup_bundle(\"\${APPS_EXPANDED}\" \"${QTPLUGINS}\" \"${DIRS}\")
-  " COMPONENT "OpenStudioApp"
-)
+# TODO: We used to run fixup_bundle on all plaforms before on 5.15.0...
+# On Ubuntu, I get: `# CMake Error at /usr/share/cmake-3.21/Modules/BundleUtilities.cmake:997 (message): error: fixup_bundle: not a valid bundle`
+if(APPLE)
+  # This does some "magic" **including on Linux** (cf #345) (On linux it will install in ./bin/ the libopenstudio.so, and libicu* stuff)
+  install(CODE "
+    # APPS might be a symbolic link which does not work with fixup_bundle
+    # expand APPS to a real path
+    get_filename_component(APPS_EXPANDED ${APPS} REALPATH)
+    include(BundleUtilities)
+    # This file defines some libs we are excluding. Some like concrt140.dll are installed already via InstallRequiredSystemLibraries
+    include(\"${CMAKE_SOURCE_DIR}/CMake/FixupBundleOverrides.cmake\")
+    fixup_bundle(\"${APPS_EXPANDED}\" \"${QTPLUGINS}\" \"${DIRS}\")
```

This is what it used to be!

```suggestion
    fixup_bundle(\"\${APPS_EXPANDED}\" \"${QTPLUGINS}\" \"${DIRS}\")
```